Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.13.3
+
+COPY scripts/entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```

Comment:
These two lines are not needed. They should be replace with what you had previously:

```
RUN apk add --no-cache git && \
    mkdir /workdir
```

Reply:
done